website/docs/reference/configs-and-properties.md

```diff
@@ -24,19 +24,22 @@ Whereas you can use **configurations** to:
 
 ## Where can I define configs?
 
-Depending on the resource type, configurations can be defined:
+Depending on the resource type, configurations can be defined in the dbt project and also in an installed package by:
 
 1. Using a [`config()` Jinja macro](/reference/dbt-jinja-functions/config) within a `model`, `snapshot`, or `test` SQL file
 2. Using a [`config` property](/reference/resource-properties/config) in a `.yml` file
 3. From the [`dbt_project.yml` file](dbt_project.yml), under the corresponding resource key (`models:`, `snapshots:`, `tests:`, etc)
 
 ### Config inheritance
 
-dbt prioritizes configurations in order of specificity, from most specificity to least specificity. This generally follows the order above: an in-file `config()` block --> properties defined in a `.yml` file --> config defined in the project file. 
+dbt prioritizes configurations in order of specificity, from most specific to least specific. This generally follows the order above: an in-file `config()` block --> properties defined in a `.yml` file --> config defined in the project file. 
 
 Note - Generic data tests work a little differently when it comes to specificity. See [test configs](/reference/data-test-configs).
 
-Within the project file, configurations are also applied hierarchically. The most specific config always "wins": In the project file, configurations applied to a `marketing` subdirectory will take precedence over configurations applied to the entire `jaffle_shop` project. To apply a configuration to a model, or directory of models, define the resource path as nested dictionary keys.
+Within the project file, configurations are also applied hierarchically. The most specific config always takes precedence. In the project file, for example, configurations applied to a `marketing` subdirectory will take precedence over configurations applied to the entire `jaffle_shop` project. To apply a configuration to a model or directory of models, define the [resource path](reference/resource-configs/resource-path) as nested dictionary keys.
+
+Configurations in your root dbt project have _higher_ precedence than configurations in installed packages. This enables you to override the configurations of installed packages, providing more control at a higher level of granularity. 
```

Comment:
Just removed the second "higher" because sometimes "more granular" is framed as lower level like in computing (low level operations) and data management (more granular data means lower level data points rather than summarized or aggregated.)
```suggestion
Configurations in your root dbt project have _higher_ precedence than configurations in installed packages. This enables you to override the configurations of installed packages, providing more control at the more granular level. 
```

Reply:
i totally understand how it can read that way but i don't think "the more granular level" is correct either.  

this is the context i'm working off of, described in the [issue](https://github.com/dbt-labs/docs.getdbt.com/issues/3239):

![Screenshot 2024-06-05 at 3 59 59 PM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/c26c4a38-e2e5-4178-b955-9257d309599c)

Reply:
maybe it's sufficient to say `This enables you to override the configurations of installed packages, providing more control over your dbt runs.` ?

Reply:
I like that!